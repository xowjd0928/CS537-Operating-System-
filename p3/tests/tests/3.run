../solution/wsh <tests/3.wsh
