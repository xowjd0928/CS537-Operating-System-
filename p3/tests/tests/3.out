wsh> hello world
wsh> 1) echo hello world
wsh> 